Library/Formula/osquery.rb

```diff
@@ -52,9 +53,9 @@ def install
   test do
     require 'open3'
     Open3.popen3("#{bin}/osqueryi") do |stdin, stdout, _|
-      stdin.write(".mode line\nSELECT major FROM osx_version;")
+      stdin.write(".mode line\nSELECT version FROM osquery_info;")
       stdin.close
-      assert_equal "major = 10\n", stdout.read
+      assert_equal "version = #{version}\n", stdout.read
```

Comment:
Isn't the version of the `osquery` package up to us? Git tags and the osquery product version are deeply engrained in how we publish packages, that won't change for us anytime soon. And if it does, won't the responsibility of identifying that be ours, as the project package maintainers?